Codecademy/gamut
# feat(Icon): add StarFilledCircleIcon

### Overview


Adds the `StarFilledCircleIcon` to `gamut-icons`


### PR Checklist

- [x] Related to designs:https://www.figma.com/file/IGWuvtcKcBcrCBg100CemB/Quizzes-%26-Practice?node-id=1844%3A26258

#### Testing Instructions

- Run storybook and verify `StarFilledCircleIcon` is shown in the regular icons section
